api/src/backend/api/authentication.py

```diff
@@ -0,0 +1,83 @@
+"""
+API Key Authentication for Prowler API.
+"""
+import logging
+
+from django.contrib.auth.models import AnonymousUser
+from django.utils import timezone
+from rest_framework import authentication, exceptions
+
+from api.models import APIKey
+
+logger = logging.getLogger(__name__)
+
+
+class APIKeyAuthentication(authentication.BaseAuthentication):
+    """
+    Simple API key authentication.
+    
+    Clients should authenticate by passing the API key in the "Authorization"
+    HTTP header, prepended with the string "ApiKey ". For example:
+    
+        Authorization: ApiKey pk_abcdef.1234567890abcdef...
+    """
+    keyword = 'ApiKey'
+    
+    def authenticate(self, request):
+        auth_header = authentication.get_authorization_header(request).decode('utf-8')
+        
+        if not auth_header:
+            return None
+            
+        try:
+            auth_type, api_key = auth_header.split(' ', 1)
```

Comment:
Is there no other choice to get the Authentication Header API Key value?

Reply:
There are other choices like Regular Expression Parsing or Manual String Slicing but the current approach 

* It's simple and reliable
* Follows HTTP authentication header conventions (Type token)
* Used by DRF's built-in authentication classes
* Handles edge cases gracefully with split(' ', 1) (only splits on first space)